### PR TITLE
main/py-django: add py3 subpackage

### PR DESCRIPTION
**Pull request by almikhayl**
main/py-django: add py3 subpackage

### CODE REVIEW

File: main/py-django/APKBUILD [hunk 1]
```diff
@@ -3,16 +3,16 @@
...
 _pkgname=Django
 pkgver=1.11
-pkgrel=0
+pkgrel=1
 pkgdesc="A high-level Python Web framework"
 url="http://djangoproject.com/"
 arch="noarch"
 license="BSD"
-depends="python2"
+depends=""
```
Comment (Shizmob) [inline main/py-django/APKBUILD]: You can omit this line if it's now empty.

File: main/py-django/APKBUILD [hunk 2]
```diff
@@ -27,12 +27,33 @@ builddir="$srcdir"/$_pkgname-$pkgver
 
 build() {
 	cd "$builddir"
-	python2 setup.py build || return 1
+	python2 setup.py --quiet build || return 1
```
Comment (Shizmob) [inline main/py-django/APKBUILD]: `|| return 1` statements can be omitted now.

File: main/py-django/APKBUILD [hunk 3]
```diff
@@ -27,12 +27,33 @@ builddir="$srcdir"/$_pkgname-$pkgver
...
+	python3 setup.py --quiet build || return 1
 }
 
 package() {
 	cd "$builddir"
-	python2 setup.py install --root "$pkgdir" --skip-build || return 1
+	python2 setup.py --quiet install --root "$pkgdir" --skip-build || return 1
```
Comment (Shizmob) [inline main/py-django/APKBUILD]: Likewise, `|| return 1` statements are unneeded.

File: main/py-django/APKBUILD (hunk 1 above)
Comment (almikhayl) [inline main/py-django/APKBUILD]: Thanks for suggestion. It seems that at least not so few packages providing py3 and py2 subpackages use empty but present `depends` variable. For example main/py-yaml, main/py-tz, py-asn1. For now I will leave it as is.

**Comment by jirutka** [comment]
Django installs `/usr/bin/django-admin`, so you’d end up with conflicting file between py2 and py3 subpackage. Please add version suffix (`-2`, `-3`) to it and symlink unsuffixed to the py3 variant.

**Pull request closed without merging**